tests/test_context_managers.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+import contextvars
+import uuid
+
+
+session_id_var = contextvars.ContextVar("session_id", default=str(uuid.uuid4()))
```

Comment:
Same here - slight preference to not set a default. Makes it more explicit and simpler to test.

Reply:
This was just for testing, if that's ok to keep